examples/acl/test/acl_test.clj

```diff
@@ -92,24 +97,50 @@
       last
       Integer/parseInt))
 
+(defn embedded-postgres!
+  [config]
+  (let [pg (.start (EmbeddedPostgres/builder))
+        pg-port (.getPort pg)
+        init-sql (slurp "./Docker/init.sql")
+        db-config (-> config
+                      :framework.db.storage/postgresql
+                      (assoc
+                        :port pg-port
+                        :subname (str "//localhost:" pg-port "/acl")))]
+    (jdbc/execute! (dissoc db-config :dbname) [init-sql])
+    (assoc config :framework.db.storage/postgresql db-config)))
+
+(defn migrate!
+  [config]
+  (let [db (:framework.db.storage/postgresql config)
+        mig-config (assoc (:framework.db.storage/migration config) :db db)]
+    (migratus/migrate mig-config))
+  config)
+
 (defn std-system-fixture
   [f]
   (let [config (config/edn)
         system (-> config
+                   embedded-postgres!
+                   migrate!
                    comps/system
                    component/start)]
     (try
-      (delete-posts)
-      (new-post "Test post")
-      (new-post "Second Test post")
       (f)
       (finally
         (delete-posts)
```

Comment:
I think it is nice to have db data persisted, in case there you want to look at them.
And either you are recreating whole db at the beginning of every test case

Reply:
I see you point!

Reply:
But if you want to see what is in the embed database you should stop the test execution with a breakpoint anyway.

Reply:
> And either you are recreating whole db at the beginning of every test case
I changed the fixture behavior to :once. Took too much time to restart the app and rebuild the database for my taste.